### PR TITLE
Kristi

### PR DESCRIPTION
**Pull request by kristi-h**
Created components for filtering based on city, state, foodtype; created Context for filters; made changes to Menu API to dynamically display rendered results; added some light css fixes for display; flattened data returned from api so array can be filtered off of any one category.

### CODE REVIEW

File: src/components/API/getMenuAPI.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import {React, useState} from 'react';
```
Comment (yusuken1121) [inline src/components/API/getMenuAPI.jsx:1]: It seems they were probably left in by mistake. If they're not necessary, I recommend removing them.

**Comment by kristi-h** [comment]
@yusuken1121 sure thing~ committed the deleted the comments so PR should be updated.

File: src/contexts/FoodtypeContext.jsx
Comment (yusuken1121) [inline src/contexts/FoodtypeContext.jsx:1]: I think FoodtypeContext.jsx is included in LocationContext.jsx. You can either delete FoodtypeContext.jsx or separate the features.

**Review by arositen — APPROVED** [review]
looks good!

File: src/contexts/FoodtypeContext.jsx
Comment (kristi-h) [reply src/contexts/FoodtypeContext.jsx:1]: it was just a test component..deleted!

**Review by MatthewNeie — APPROVED** [review]

**Pull request closed without merging**